addons/core/translations/resources/en-us.yaml

```diff
@@ -444,6 +444,8 @@ worker:
   table:
     session_count: Session Count
     ip_address: IP Address
+  dropdown:
+    delete: Remove Worker
```

Comment:
I think this could've just been added to the existing `actions` for workers, we both had an action in our PRs

Reply:
@ZedLi incoming